docs/cql/cqlsh.rst

```diff
@@ -6,18 +6,22 @@
 CQLSh: the CQL shell
 --------------------
 
-cqlsh is a command line shell for interacting with Cassandra through CQL (the Cassandra Query Language).  It is shipped
-with every Cassandra package and can be found in the bin/ directory alongside the Cassandra executable.  cqlsh utilizes
+cqlsh is a command line shell for interacting with ScyllaDB through CQL (the Cassandra Query Language). It is shipped
+with every ScyllaDB package and can be found in the ``bin/`` directory. 
+In addition, it is available on `Docker Hub <https://hub.docker.com/r/scylladb/scylla-cqlsh>`_ 
+and in the `Python Package Index (PyPI) <https://pypi.org/project/scylla-cqlsh/>`_.
```

Comment:
maybe also point to https://github.com/scylladb/scylla-cqlsh, where there few examples and instruction on how to use those ?

or copy one or two examples from there ?